crates/rustc_codegen_spirv/src/linker/new_structurizer.rs

```diff
@@ -168,7 +168,12 @@ impl Structurizer<'_> {
             let block_id = self.func.blocks()[block].label_id().unwrap();
             let terminator = self.func.blocks()[block].instructions.last().unwrap();
             let mut region = match terminator.class.opcode {
-                Op::Return | Op::ReturnValue | Op::Kill | Op::Unreachable => Region {
+                Op::Return
+                | Op::ReturnValue
+                | Op::Kill
+                | Op::IgnoreIntersectionKHR
+                | Op::TerminateRayKHR
+                | Op::Unreachable => Region {
```

Comment:
There's a comment in this file that goes like this:
```rust
    /// The `merge` block must be terminated by one of `OpReturn`, `OpReturnValue`,
    /// `OpKill`, or `OpUnreachable`.
```

Can you update it, or at least make the phrasing less complete (i.e. turn those specific instructions into examples more than an exhaustive list)?